src/main/java/org/jenkins/plugins/statistics/gatherer/util/PropertyLoader.java

```diff
@@ -82,7 +82,7 @@ public static String getEnvironmentProperty(
     }
 
     public static boolean isTrue(String value) {
-        return value != null ? value.toLowerCase().equals("true") : false;
+        return value != null && value.toLowerCase().equals("true");
```

Comment:
Use `Boolean.parseBoolean(value)`, does the job and checks for nulls also

Reply:
If you are fine that `parseBoolean()` is case-insensitive - e.g. value "tRuE" is also evaluated to `true` than I can do that.